src/site/content/en/blog/wakelock/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Stay awake with the WakeLock API
+subhead: The Wake Lock API provides a way to prevent the device from dimming or locking the screen when an application needs to keep running.
+authors:
+  - petelepage
+  - thomassteiner
+description: To avoid draining the battery, most devices will quickly fall asleep when left idle. While this is fine for most of the time, there are some applications that need to keep the screen or the device awake in order to complete some work. The Wake Lock API provides a way to prevent the device from dimming or locking the screen or prevent the device from going to sleep when an application needs to keep running.
+date: 2018-12-18
+updated: 2019-10-10
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - capabilities
+  - wake-lock
+hero: hero.jpg
+alt: Sleeping kowla
+draft: true
+---
+
+{% Aside %}
+  The Wake Lock API, is part of
+  [capabilities project](https://developers.google.com/web/updates/capabilities),
+  and is currently in development, this post will be updated as the
+  implementation progresses.
+{% endAside %}
+
+## What is the Wake Lock API? {: #what }
+
+To avoid draining the battery, most devices quickly go to sleep when left
+idle. While this is fine most of the time, some applications need to keep the
+screen or the device awake in order to complete their work. For example, a
+run-tracking app (turns the screen off, but keeps the system awake), or a game,
+like [Ball Puzzle](https://ball-puzzle.appspot.com/), that uses the device
+motion APIs for input.
+
+The [Wake Lock API][spec-ed] provides a way to prevent the device from dimming
+and locking the screen or prevent the device from going to sleep. This
+capability enables new experiences that, until now, required a native app.
+
+The Wake Lock API aims to reduce the need for hacky and potentially
+power-hungry workarounds. It addresses the shortcomings of an older API
+which was limited to simply keeping the screen on, and had a number of
+security and privacy issues.
+
+### Suggested use cases for the Wake Lock API {: #use-cases }
+
+[RioRun](https://www.theguardian.com/sport/2016/aug/06/rio-running-app-marathon-course-riorun),
+a web app developed by [The Guardian](https://www.theguardian.com/)
+that takes you on a virtual audio tour of Rio, following the route of the 2016
+Olympic marathon would be a perfect use case. Without wake locks, your screen
+will turn off frequently, making it hard to use.
+
+Of course, there are plenty of others:
```

Comment:
'others' to 'other use cases'.

The antecedent to this is buried two sentences back and might be a little unclear.